public_routing.tf

```diff
@@ -0,0 +1,29 @@
+#-------------------------------------------------------------------------------
+# Set up routing in the VPC for the public subnets, which use the
+# VPC's default routing table.
+#
+# The routing for the private subnets is configured in
+# private_routing.tf.
+# -------------------------------------------------------------------------------
+
+# Default route table (used by public subnets)
+resource "aws_default_route_table" "public" {
+  default_route_table_id = aws_vpc.the_vpc.default_route_table_id
+  tags                   = var.tags
+}
+
+# Route all non-local COOL (outside this VPC but inside the COOL)
+# traffic this VPC through the transit gateway
```

Comment:
I think you meant to delete "this VPC" from this comment?

Reply:
Fixed in commit 9c5e061.